src/umbraco.businesslogic/UserGroup.cs

```diff
@@ -0,0 +1,309 @@
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using Umbraco.Core;
+using Umbraco.Core.Events;
+
+namespace umbraco.BusinessLogic
+{
+    /// <summary>
+    /// Represents a umbraco Usergroup
+    /// </summary>
+    [Obsolete("Use the UserService instead")]
```

Comment:
Wat!? @AndyButland  you created a new obsolete legacy class? why? I'm just removing this now since we don't need it. maybe you were just being extra thorough?